src/pages/Home.js

```diff
@@ -31,47 +40,112 @@ const Home = () => {
         setAddedProject(false)
     }
 
-    const postNewProject = () => {
+    const postProject = () => {
         Fire
         .firestore()
         .collection("projects")
         .add({
             projectName: projectName,
             userId: user.uid,
         })
-        .then(() => console.log('succes'))
+        .then()
         .catch(error => console.error("Error adding document: ", error))
         setAddedProject(true)
         dispatch(fetchProjects(user.uid));
```

Comment:
gebeurd er na een post hier meteen een get weer om alle data op te halen ?

Reply:
Met GraphQL krijg je meteen de gemuteerde data terug. dus heb je maar 1 round trip, plus kan je selecteren welke data belangrijk voor jou is. op de plek waar je de mutatie doet. Dus dan fetch je ook niet te veel.